services/web-server/src/login/strategies/github.js

```diff
@@ -127,10 +131,10 @@ module.exports = class Github {
           scope: 'repo',
         },
         async (accessToken, refreshToken, profile, next) => {
-          await this.GithubAccessToken.create({
-            userId: profile.id,
+          await this.db.fns.add_github_accesstoken(
+            profile.id,
             accessToken,
```

Comment:
...and encrypted here

Reply:
Indeed, I totally forgot about encryption, good that you checked so carefully! ❤️ 
Done now.